Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -20,6 +20,7 @@
 
 #include <GraphMol/SmilesParse/SmilesParse.h>
 #include <GraphMol/RDKitQueries.h>
+#include <GraphMol/StereoGroup.h>
```

Comment:
StereoGroup.h should also be added to `rdkit_headers` in `Code/GraphMol/CMakeLists.txt`
(I mentioned this in a comment already but am including it here too so that it shows up in the review)